Orange/evaluation/testing.py

```diff
@@ -202,6 +202,53 @@ def get_fold(self, fold):
 
         return results
 
+    def get_table_with_probabilities(self, model_names):
+        """
+            Return the Orange.data.Table with data, predictions and probabilities (if the task is classification).
```

Comment:
The standard is for docstrings to be aligned with opening tri-quotes (cf. e.g. `np.hstack()`).